### PR TITLE
Dark-mode-profile

### PR DESCRIPTION
**Pull request by wildcraft958**
1. [FEATURE] Implement darkmode functionality in /profile route #52
has been added.
2. Little bit of code cleaning

### CODE REVIEW

File: public/js/Profile.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ button.addEventListener('click', () => {
   if (body.classList.contains('light-theme')) {
     body.classList.remove('light-theme');
     body.classList.add('dark-theme');
-    button.innerHTML = '<i class="fas fa-sun"></i>'; // Change icon for dark theme
+    button.innerHTML = '<i class="fas fa-sun"></i>';
```
Comment (sristy17) [inline public/js/Profile.js:8]: donot remove the comments

File: views/pages/profile.ejs [hunk 2]
```diff
@@ -11,178 +10,115 @@
...
-<link rel="icon" href="/images/favicon.ico" type="image/x-icon">
-<!-- Font Awesome CDN -->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" />
-
-<!-- Tailwind css cdn  -->
-<script src="https://cdn.tailwindcss.com"></script>
-</head>
-
-<body class="light-theme" id="body">
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet" />
```
Comment (sristy17) [inline views/pages/profile.ejs:39]: what did u do with other fonts?

File: views/pages/profile.ejs [hunk 3]
```diff
@@ -11,178 +10,115 @@
...
-
+            <input type="password" id="password" name="password" class="w-full p-3 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500 transition-shadow duration-200" placeholder="Enter your password" />
           </div>
-
-          <!-- Submit Button -->
-          <button type="submit"
-            class="w-full bg-[#7C3AED] text-white py-3 rounded-lg font-semibold hover:ring-opacity-80 active:bg-[#4C1D95] hover:shadow-lg transition-all duration-300 focus:ring-4 focus:ring-purple-300 focus:ring-opacity-50">
-            Submit
-          </button>
+          <button type="submit" class="w-full bg-purple-700 text-white py-3 rounded-lg font-semibold hover:ring-opacity-80 active:bg-purple-900 hover:shadow-lg transition-all duration-300 focus:ring-4 focus:ring-purple-300">Submit</button>
```
Comment (sristy17) [inline views/pages/profile.ejs:85]: comment?

File: views/pages/profile.ejs [hunk 4]
```diff
@@ -11,178 +10,115 @@
...
+            <label for="signIn-password" class="block text-gray-700 font-medium mb-2">Password</label>
+            <input type="password" id="signIn-password" name="password" class="w-full p-3 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500 transition-shadow duration-200" placeholder="Enter your password" />
           </div>
-
-          <!-- Submit Button -->
-          <button type="submit"
-            class="w-full bg-[#7C3AED] text-white py-3 rounded-lg font-semibold hover:ring-opacity-80 active:bg-[#4C1D95] hover:shadow-lg transition-all duration-300 focus:ring-4 focus:ring-purple-300 focus:ring-opacity-50">
-            Submit
-          </button>
+          <button type="submit" class="w-full bg-purple-700 text-white py-3 rounded-lg font-semibold hover:ring-opacity-80 active:bg-purple-900 hover:shadow-lg transition-all duration-300 focus:ring-4 focus:ring-purple-300">Submit</button>
```
Comment (sristy17) [inline views/pages/profile.ejs:102]: we only commented in some specific parts no need to remove or change them

File: public/css/modal.css [hunk 5]
```diff
@@ -93,6 +93,7 @@
       transform: rotate(360deg);
     }
   } 
+<<<<<<< HEAD
```
Comment (sristy17) [inline public/css/modal.css:96]: update ur commits then push the pr

**Comment by sristy17** [comment]
> 1. [FEATURE] Implement darkmode functionality in /profile route [[FEATURE] Implement darkmode functionality in `/profile` route #52](https://github.com/sristy17/MindDrive/issues/52)
>    has been added.
> 2. Little bit of code cleaning

hey there are many conflicts we cannot merge the pr i am closing this update your code take a pull then create a new pr, do not do code clean up we will do it just make the required changes, also make the card bg also dark make it suitable

**Comment by wildcraft958** [comment]
sure

**Pull request closed without merging**